packages/flutter/lib/src/widgets/animated_switcher.dart

```diff
@@ -48,9 +48,14 @@ typedef Widget AnimatedSwitcherTransitionBuilder(Widget child, Animation<double>
 /// Signature for builders used to generate custom layouts for
 /// [AnimatedSwitcher].
 ///
-/// The function should return a widget which contains the given children, laid
-/// out as desired. It must not return null.
-typedef Widget AnimatedSwitcherLayoutBuilder(List<Widget> children);
+/// The builder should return a widget which contains the given children, laid
+/// out as desired. It must not return null. The builder should be able to
+/// handle an empty list of previousChildren, or a null currentChild.
+///
+/// The previousChildren list is an unmodifiable list, sorted with the oldest
+/// at the beginning and the newest at the end. It does not include the
+/// currentChild.
+typedef Widget AnimatedSwitcherLayoutBuilder(Widget currentChild, List<Widget> previousChildren);
```

Comment:
argument names in these paragraphs should be marked with backticks.

Reply:
Done.